builtin/providers/openstack/resource_openstack_compute_floatingip_v2.go

```diff
@@ -0,0 +1,114 @@
+package openstack
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/rackspace/gophercloud/openstack/compute/v2/extensions/floatingip"
+)
+
+func resourceComputeFloatingIPV2() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceComputeFloatingIPV2Create,
+		Read:   resourceComputeFloatingIPV2Read,
+		Update: nil,
+		Delete: resourceComputeFloatingIPV2Delete,
+
+		Schema: map[string]*schema.Schema{
+			"region": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				DefaultFunc: envDefaultFunc("OS_REGION_NAME"),
+			},
+
+			"pool": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				DefaultFunc: envDefaultFunc("OS_POOL_NAME"),
+			},
+
+			// exported
+			"id": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
You shouldn't need to declare this - every resource gets an `id` field by default.